sdk/cosmos/azure_data_cosmos/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+* `FeedPager<T>` now asynchronously iterates items of type `T` instead of pages containing items of type `T`. Call `Pager::into_pages()` to get a `PageIterator` to asynchronously iterate over all pages.
```

Comment:
The type name is wrong now that `Pager` is gone, but I think we can just remove the type name here anyway.

```suggestion
* `FeedPager<T>` now asynchronously iterates items of type `T` instead of pages containing items of type `T`. Call `into_pages()` to get a `PageIterator` to asynchronously iterate over all pages.
```

Reply:
Given my reply to your comment below - that `Pager` still exists - isn't the original text still valid?